packages/hdwallet-ledger/src/utils.ts

```diff
@@ -96,503 +95,277 @@ export const createXpub = (
 };
 
 type NetworkMagic = {
-  apiName: string;
-  unit: string;
-  name: string;
-  appName?: string;
-  satoshi?: number;
   bitcoinjs: {
-    bech32?: string;
     bip32: {
-      private?: number;
       public: Partial<Record<core.BTCInputScriptType, number>>;
     };
-    messagePrefix: string;
-    pubKeyHash?: number;
-    scriptHash?: number;
-    wif?: number;
   };
   sigHash?: number;
   isSegwitSupported?: boolean;
-  handleFeePerByte?: boolean;
-  additionals?: string[];
```

Comment:
We may not use them now and probably never in the future, but would keep it just to avoid silently breaking things - according to grepping ledger repos, this is perfectly valid (though possibly outdated and not relevant anymore, but who knows...) https://github.com/LedgerHQ/ledger-wallet-webtool/blob/2db847a27e42b2353e979240cc711ad8fb46fe71/src/Networks.js#L159

Reply:
Regardless of whether or not it is used/anymore, it does conforms to *some* schema (no matter how outdated it may be), which confirms we probably shouldn't clean this up to keep on honoring said schema

Reply:
On second thought: meh